src/services/insight.js

```diff
@@ -62,3 +62,29 @@ export const getAddressBalanceFromNode = (address: string) => {
   })
     .then(validateResponse('getAddressBalanceFromNode'));
 };
+
+export const getBTCTransactionsFromNode = (address: string) => {
+  return fetch(`${BITCOIN_INSIGHT_URL}/address/${address}/txs?limit=0`, {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+    },
+  }).then(validateResponse('getBTCTransactionsFromNode'))
+    .then(response => response.json())
+    .then(txs => {
+      const fullTxs = txs.map(e => {
+        return fetch(`${BITCOIN_INSIGHT_URL}/tx/${e.mintTxid}/populated`, {
```

Comment:
Are we gonna fetch all transactions and then fetch the transaction details for each one? I think this could become a problem quite quickly :thinking:

Reply:
we will need the info from the populated tx info to form the current way of presenting transactions. The best way would be to form and host the bitcore node where we could bind the information and build our own api.